lib/Basics/FileUtils.h

```diff
@@ -154,4 +164,16 @@ std::string dirname(std::string const&);
 // returns the output of a program
 std::string slurpProgram(std::string const& program);
 
+// returns the output of a program
+std::string slurpProgramWithArgs(std::string const& program,
+                                 std::vector<std::string> args);
+
```

Comment:
I think in the changes in FileUtils.cpp, this function has not been added, so it probably can be removed from here now.
```suggestion
```